api/server/structs/conversions_block.go

```diff
@@ -2519,6 +2520,7 @@ func (b *BeaconBlockContentsElectra) ToConsensus() (*eth.BeaconBlockContentsElec
 	}, nil
 }
 
+// nolint:gocognit
```

Comment:
Instead of dismissing the linter. Can you refactor this so it's not so complex?

Reply:
The method is basically only boilerplate. We should generate it in the future, hopefully Kasey's library will allow this (this is the plan as far as I know). I would prefer to keep it as straightforward as possible, even if this means it gets too long.